dpgen2/exploration/render/traj_render_lammps.py

```diff
@@ -63,6 +63,8 @@ def get_confs(
         id_selected: List[List[int]],
         type_map: Optional[List[str]] = None,
         conf_filters: Optional["ConfFilters"] = None,
+        use_ele_temp: int = 0,
```

Comment:
provide as class attribute?

Reply:
I'm not sure if I understand it. Set `use_ele_temp` as class attribute and update it in `__init__` function?